versioned_docs/version-2.x/magic-button/contribution-mode.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 1
+title: Contribution Mode
+description:
+  Contribution Mode is the core of the Magic Button tool. When Contribution
+  Mode is enabled for a user, your storefront displays a set of tools created
+  for content creators.
+---
+
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="2.25" />
+
+<p>{frontMatter.description}</p>
+
+## How to activate Contribution Mode
+
+### Activate with Config Provider
+
+The **Magic Button** is only visible when the **Contribution Mode** is active.
+
+The default activation for **Contribution Mode** is defined by the
+[`contributionModeConfigProvider`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/2cc02f42cf4f984f3269f4b1bfc07cea9a53795f/src/server/modules/front-commerce/contribution-mode/contributionModeConfigProvider.js).
+By default, the contribution mode is controlled by the `FRONT_COMMERCE_CONTRIBUTION_MODE_FORCE` environment variable.
+
+You can follow the
+[Override an existing configuration](/docs/2.x/advanced/server/configurations#override-an-existing-configuration)
+to implement your own rule to conditionally enable the contribution mode.
```

Comment:
**nit:** This sentence feels like it's missing a word.

**suggestion:** either remove "the"
```suggestion
You can follow [Override an existing configuration](/docs/2.x/advanced/server/configurations#override-an-existing-configuration)
to implement your own rule to conditionally enable the contribution mode.
```
or add a word after

```suggestion
You can follow the
[Override an existing configuration guide](/docs/2.x/advanced/server/configurations#override-an-existing-configuration)
to implement your own rule to conditionally enable the contribution mode.
```